include/dirac_quda.h

```diff
@@ -542,7 +542,6 @@ namespace quda {
     DiracWilson(const DiracWilson &dirac);
     DiracWilson(const DiracParam &param, const int nDims); // to correctly adjust face for DW and non-deg twisted mass
 
-    virtual ~DiracWilson();
```

Comment:
Just out of interest, why are we removing the virtual destructors? Is there a lower base class? Or did we make them protected or something?

Reply:
Since this was a trivial destructor, we're deferring to the compiler to generate a default destructor.  I think this is best practice.